### ISSUE TITLE
Performance of VIT-B/32 is worse than RN50 on YFCC15M

### ISSUE THREAD

**Issue by someone-1024** [mlfoundations/open_clip#39]
We are trying to re-implement CLIP ViT-B/32 pre-trained on YFCC15M provided by OpenAI. But our result is lower than RN50 reported by the paper and your repo (still under training, but almost finished, current ImageNet zero-shot accuracy is around 27% - 28%). So we wonder if you have tried to train a ViT-B/32 on YFCC? Do you have the same finding? Thanks.

**Comment by mitchellnw** [comment]
Yes, we found the same thing. We believe that ViT > RN in the high data (~400m) regime.

**Comment by rwightman** [comment]
@someone-1024 see also https://github.com/mlfoundations/open_clip/discussions/56 ... the same is true for cc12m, the ViT models require more data to pass RN50 models.

I hope to do a yfcc + cc12m training at some point to see if that is enough to have ViT-B-32 at parity with RN50. I'll update in the linked discussion above when I have any result.
